src/index.md

```diff
@@ -1307,6 +1307,40 @@ The fix for the main issue should remove the test from the ProblemList or remove
 
 After a failure is handled by excluding a test, the main JBS issue should be re-triaged and possibly given a new priority. This should be handled by the standard triage process. A test exclusion results in an outage in our testing. This outage should be taken into consideration when triaging, in addition to the impact of the bug itself.
 
+# Adding new code in Hotspot
```

Comment:
HotSpot should be written with capital S. More occurrences later.

I wonder if we should call this section something more generic, like "HotSpot Development" so that it can host more than just addition of code.